### ISSUE TITLE
ppc64le/orientdb docker image upload to dockerhub

### ISSUE THREAD

**Issue by ghatwala** [orientechnologies/orientdb-docker#29]
Hi Guys,

I wanted to know if you'll will be uploading the docker image to dockerhub for ppc64le. I had a commit for the same dockerfile here : https://github.com/orientechnologies/orientdb-docker/commit/a054d9aa0c312087b569b19271c95b22619d6121

Let me know the plan ahead to upload ppc64le/orientdb docker image to dockerhub similar to this : https://hub.docker.com/_/orientdb/

Regards,
Amit

**Comment by robfrank** [comment]
Dockerfile for ppc64 is present in this repository and updated at every release. 
AFAIK dockerhub hosts x86_64 images as official ones. 
I'll investigate if we can add to our official images file the ppc64 one.

**Comment by ghatwala** [comment]
yes true , dockerhub has orientdb-docker image which is for x86_64 only. However for ppc64le platform related, all application related docker images  are kept here at dockerhub : https://hub.docker.com/u/ppc64le/ .

We could then officially add the docker image for orientdb-ppc64le at above location . 

Let me know your thoughts ?

**Comment by robfrank** [comment]
Hi, but I can't push anything on ppc64le, it is up to the user named **ppc64u** to push it.
I can push something like _orientb/orientdb:2.2.17-ppc64le_ 
I will ask docker if they handle official images based on ppc64 . 
Official images are handled from docker itself through this repo: https://github.com/docker-library/official-images

**Comment by ghatwala** [comment]
Ok , thanks for your update !!

**Comment by vaibhavsood** [comment]
Hi, want to check back on this. I'm looking at expanding the docker-library official-images for multiple architectures, orientdb has a ppc64le dockerfile but does not publish official ppc64le images

To do that, need to update https://github.com/docker-library/official-images (as noted earlier by @robfrank )

Specifically, need to update this file https://github.com/docker-library/official-images/blob/master/library/orientdb, 

changes will be similar to this: https://github.com/docker-library/official-images/pull/3436/commits/0155f8079290dce6bdd1dae96ef6c55ff9ce0de3  (for orientdb for now add ppc64le in Architectures)

I am planning to submit a PR to official-images with these changes, want to check if it will be okay and if there are any comments/suggestions? thanks!

**Comment by robfrank** [comment]
GREAT, please submit!  Should I change the structure of this repo to enable ppc64?

**Comment by vaibhavsood** [comment]
Hi @robfrank, 

I had opened a PR on official-images for multi-arch orientdb https://github.com/docker-library/official-images/pull/3480

This was blocked/closed since the base image alpine was not multi-arch which has subsequently been fixed

As per @tianon, they recommend having a single dockerfile for all arch's, so want to request merging of the ppc64le changes into the intel dockerfile (currently the ppc dockerfile is separate from the intel one)

Can this be done so that can move ahead with multi-arching this docker image? thanks!!

**Comment by vaibhavsood** [comment]
Hi @robfrank, want to check if i can get help with this, thanks!
